src/main/java/sysc4806/ProjectController.java

```diff
@@ -53,15 +59,11 @@
 
     @GetMapping(path="/deleteAll")
     public @ResponseBody String deleteAllProjects () {
-        projectRepo.findAll().forEach(project -> projectRepo.delete(project));
+        projectRepo.deleteAll();
```

Comment:
so the reason i put it the other way, was because i found that calling this would also delete the profs, which should not happen. can you make a test case for it?